backend/Recodo.Common/Dtos/User/AvatarDTO.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Recodo.Common.Dtos.User
+{
+    public class AvatarDTO
+    {
+        public string thumb_url { get; set; }
```

Comment:
Don't use snake case for C# properties. Instead make it `ThumbUrl` and add [JsonProperty] attribute